CREDITS.md

```
@@ -54,78 +56,50 @@ https://github.com/iconify/iconify
 There is no license specified in this repository. Please visit the source and contact the repository's owners to find out more.
 
+
 -------------------------------------------------------------------------------
 
 ## Project
-@storybook/test
+blurhash
 
 ### Source
-https://github.com/storybookjs/storybook
+https://github.com/woltapp/blurhash/tree/master/TypeScript
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)